WeWatch/WeWatch/Splash/SplashViewModel.swift

```diff
@@ -0,0 +1,106 @@
+////
+////  SplashView.swift
+////  WeWatch
+////
+////  Created by Anton on 18/12/2024.
+////
+//
+import Foundation
+
+@MainActor
+class SplashViewModel: ObservableObject {
+    
+    internal enum viewJwtError: Error {
```

Comment:
1. Enum names should start with a capital letter. 
2. Rename this to JwtViewError